src/app/@ansyn/ansyn/app-effects/effects/filters.app.effects.ts

```diff
@@ -184,6 +190,9 @@ export class FiltersAppEffects {
 	@Effect({ dispatch: false })
 	filteredOverlaysChanged$: Observable<any> = this.store$.select(selectFilteredOveralys)
 		.withLatestFrom(this.store$.select(filtersStateSelector), this.store$.select(selectOverlaysMap))
+		.filter(([filteredOverlays, filterState, overlays]: [string[], IFiltersState, Map<string, Overlay>]) => {
+			return overlays.size > 0;
```

Comment:
What happens if the current search returns an empty set?

Reply:
works fine